cves/CVE-2013-4310.yml

```diff
@@ -177,8 +189,8 @@ lessons:
     applies:
     note:
   security_by_obscurity:
-    applies:
-    note:
+    applies: true
+    note: Add obscurity for temporary fixes when addressing them later.
```

Comment:
Based just on reading this yml, I'm not sure how security by obscurity applies here. I think this should be gone into a little more in depth.

Reply:
I have added some detail for clarity. I believe the commit named "Disables DMI" is a quick fix for this vulnerability that adds some code near where the vulnerability was, some of it looking obscure yet also adding some kind of difficult to read validation and sanitation.